cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -754,43 +754,53 @@ func (c *client) WaitForResourceStatuses() error {
 		klog.V(2).Info("Waiting for Cluster API resources to have statuses...")
 		clusters, err := c.clientSet.ClusterV1alpha1().Clusters("").List(metav1.ListOptions{})
 		if err != nil {
+			klog.V(10).Infof("error list clusters:", err)
```

Comment:
This message seems a bit confusing, since they are logged at Info level, but start with "error". Maybe something along the lines of: `Infof("retrying, failed to list clusters: ", err)` instead?

Reply:
works for me ~, thanks~